### PR TITLE
Add empty results fallback for Result.repetitions

### PR DESCRIPTION
**Pull request by vtomole**
Fixes: https://github.com/quantumlib/Cirq/issues/3852

### CODE REVIEW

File: cirq/study/result.py [hunk 1]
```diff
@@ -150,8 +150,11 @@ def measurements(self) -> Dict[str, np.ndarray]:
 
     @property
     def repetitions(self) -> int:
+        measurement_values = self.measurements.values()
```
Comment (balopat) [inline cirq/study/result.py:153]: can self.measurement itself be `None`?

**Review by balopat — CHANGES_REQUESTED** [review]
Thanks for jumping in - added a nit

**Comment by balopat** [comment]
I'm gonna press the update branch button to nudge Github Actions

File: cirq/study/result.py (hunk 1 above)
Comment (vtomole) [reply cirq/study/result.py]: Added check for this. :+1:

File: cirq/study/result_test.py [hunk 2]
```diff
@@ -158,6 +158,16 @@ def test_multi_measurement_histogram():
         }
     )
 
+    result = cirq.Result.from_single_parameter_set(
+        params=cirq.ParamResolver({}),
+        measurements={},
+    )
```
[the remark this answers is not in the document]
Comment (vtomole) [reply cirq/study/result_test.py]: Yup. I'm just following the same pattern that every assert in this `def test_multi_measurement_histogram` follows. Just modified the test.

File: cirq/study/result_test.py [hunk 3]
```diff
@@ -158,6 +158,16 @@ def test_multi_measurement_histogram():
         }
     )
```
[the remark this answers is not in the document]
Comment (vtomole) [reply cirq/study/result_test.py]: Done

File: cirq/study/result.py [hunk 4]
```diff
@@ -150,6 +150,8 @@ def measurements(self) -> Dict[str, np.ndarray]:
 
     @property
     def repetitions(self) -> int:
+        if not self.measurements or not self.measurements.values():
```
Comment (balopat) [inline cirq/study/result.py:153]: I think the empty dict is Falsey so we could just use this: 
```suggestion
        if not self.measurements:
```

**Review by balopat — APPROVED** [review]
LGTM with the last nit to simplify the if statement

File: cirq/study/result.py (hunk 4 above)
Comment (vtomole) [reply cirq/study/result.py]: Ah. Thanks.

**Pull request merged**